client-java/controller/src/main/java/org/evomaster/client/java/controller/internal/db/constraint/TableConstraintExtractor.java

```diff
@@ -0,0 +1,14 @@
+package org.evomaster.client.java.controller.internal.db.constraint;
+
+import org.evomaster.client.java.controller.api.dto.database.schema.DbSchemaDto;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.List;
+
+public abstract class TableConstraintExtractor {
```

Comment:
Add a comment specifying that constraints are handled differently in different databases (there is no standard for it) and JDBC does not support it, and so we need customize extractors for each single database we want to support

Reply:
Thanks. I added the required comments.